local_extensions/__init__.py

```diff
@@ -11,19 +12,57 @@
 
 
 VOLTO_MIN_VERSION = 16
-VOLTO_GENERATOR_MIN_VERSION = 6
+
+
+VOLTO_GENERATOR_VERSIONS = {
+    16: (6, 8),
+    17: (8, 9),
```

Comment:
And this one should be 
```suggestion
    17: (7, 9),
```
no?